### DISCUSSION TITLE
Improving examples

### DISCUSSION THREAD

**Discussion by simonschwaer** [groupmm/playplot#2] [Ideas]
The example notebooks are super helpful for me, but it would be nice if they would be executable "out-of-the-box". Maybe they can be stripped down to show the same functionality but with minimal external data required?

I can update them and make a PR when it's a good time for that. Or will they be changed anyway?

**Comment by daniel-keitel** [comment]
The required audio/data files will be included in the repository for all examples except ``advanced``.

It may be a good idea to change the examples , so that if they can't find the example data, they automatically download all required files.
This way one could download only a single example directly from github and run it without needing to clone it.
Caveat: The examples get more bulky with more initialization code at the top.
